docs/developer/extending/apps/developing-with-tunnels.mdx

```diff
@@ -0,0 +1,106 @@
+---
+title: Tunneling Apps
+---
+
+## What is a tunnel?
+
+The Saleor extendability model relies on the concept of Apps. If you want to read more about them, head to [the Apps page](developer/extending/apps/key-concepts.mdx).
+
+As you want Apps to react to events in your store, you reach for webhooks. However, webhooks pose a unique challenge: they can only communicate with public clients. If so, how are you supposed to develop your app on `localhost`?
```

Comment:
Before even webhooks can be triggered, the user will not be able to install the application without the tunnel. Since it's going to be a first problem for the app developer, maybe it's a better example than webhooks? (or use them both)